maintainers/scripts/update.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+import concurrent.futures
+import json
+import os
+import subprocess
+import sys
+
+def run_update_script(package):
+    print(f" - {package['name']}: UPDATING ...", file=sys.stderr)
+
+    subprocess.run(package['updateScript'], stdout=subprocess.PIPE, stderr=subprocess.STDOUT, check=True)
```

Comment:
```suggestion
    subprocess.run(package['updateScript'], stdout=subprocess.PIPE, stderr=subprocess.STDOUT, check=True, shell=True)
```
Currently it is possible to not just pass in a script, but also arguments.

Reply:
Note this solved the issue I had.

Reply:
Would not it be better to add shebang to the `update-python` script?

Reply:
That's another way. In any case, the interface needs to be documented.

Reply:
Just to be clear, I am not saying we should have it with arguments. If we need to create a derivation for every update we may as well have arguments in there.

Reply:
Allowing lists will allow us to avoid needing derivations per package.